great_ape_safe/ape_api/aave.py

```diff
@@ -147,14 +147,13 @@ def lever_up(self, collateral_token, borrow_token, perc):
         # borrow
         self.borrow(borrow_token, we_want_to_borrow)
 
-        # swap borrowed for more `collateral_token` and deposit back into aave
-        # TODO: add swap functions to uni_v3 class and use that instead
-        self.safe.init_sushi()
-        to_reinvest = self.safe.sushi.swap_tokens_for_tokens(
-            borrow_token,
-            we_want_to_borrow,
-            [borrow_token, registry.eth.treasury_tokens.WETH, collateral_token]
-        )[-1]
+        self.safe.init_uni_v3()
+        weth = self.safe.contract(registry.eth.treasury_tokens.WETH)
+        to_reinvest = self.safe.uni_v3.swap(
+            [borrow_token, weth, collateral_token],
+            we_want_to_borrow
+        )
+
```

Comment:
```suggestion
        # swap borrowed for more `collateral_token` and deposit back into aave
        self.safe.init_uni_v3()
        to_reinvest = self.safe.uni_v3.swap(
            [borrow_token, collateral_token], we_want_to_borrow
        )
```
no need to go through weth here! we did that on sushi due to it not having a direct wbtc/usdc pool but that is very different on univ3 :)

Reply:
good point